### DISCUSSION TITLE
post/2022-01-13-blockchain-xiaozhen

### DISCUSSION THREAD

**Discussion by giscus** [zcong1993/blog#54] [Announcements]
# post/2022-01-13-blockchain-xiaozhen

学习区块链总结了一些基础知识.

https://blog.cong.moe/post/2022-01-13-blockchain-xiaozhen

**Comment by kbrx93** [comment]
之前对于区块链的了解基本来自 https://youtu.be/bBC-nXj3Ng4。

文章很详实，知其所以然也是一种乐趣。感谢分享 👍🏻。
